Snowflake.Data/Client/SnowflakeDbCommand.cs

```diff
@@ -124,13 +128,15 @@ public override void Cancel()
         
         public override int ExecuteNonQuery()
         {
+            logger.Debug($"ExecuteNonQuery, command: {CommandText}");

```

Comment:
can you explain what `$` prefix mean?

Reply:
looks like format string?

Reply:
It is just .net syntax. Prefix `$` means content in curly bracket will be replaced by actual variable `CommandText`